UI/src/app/components/panel/panel-target.component.html

```diff
@@ -0,0 +1,109 @@
+<mat-expansion-panel (opened)="onOpen()" (closed)="onClose()" [expanded]="concept.expanded">
+  <mat-expansion-panel-header
+    #exppanelheader
+    class="mat-expansion-panel-header panel-header-{{ table.name }}"
+  >
+    <mat-panel-title>
+      <mat-icon
+        (click)="openSampleDataDialog($event)"
+        matTooltip="Open sample data"
+      >
+        list
+      </mat-icon>
+      <span>Concept</span>
+      <div class="linkSimilarItems">
+        <mat-slide-toggle
+          [disabled]="!concept.rows.length"
+          (change)="onCheckboxChange($event)"
+          (click)="$event.stopPropagation();"
+        >Link similar fields</mat-slide-toggle>
+      </div>
+    </mat-panel-title>
+  </mat-expansion-panel-header>
+  <app-panel-table
+    #conceptPanel
+    [table]="concept"
+    [tabIndex]="tabIndex"
+    [displayedColumns]="[
+      'column_indicator',
+      'column_name',
+      'column_type',
+      'comments'
+    ]"
+    (openTransform)="onOpenTransfromDialog($event)"
+  >
+  </app-panel-table>
+</mat-expansion-panel>
+<mat-expansion-panel (opened)="onOpen()" (closed)="onClose()" [expanded]="common.expanded">
+  <mat-expansion-panel-header
+    #exppanelheader
+    class="mat-expansion-panel-header panel-header-{{ table.name }}"
+  >
+    <mat-panel-title>
+      <mat-icon
+        (click)="openSampleDataDialog($event)"
+        matTooltip="Open sample data"
+      >
+        list
+      </mat-icon>
+      <span>Common</span>
+      <div class="linkSimilarItems">
+        <mat-slide-toggle
+          [disabled]="!common.rows.length"
+          (change)="onCheckboxChange($event)"
+          (click)="$event.stopPropagation();"
+        >Link similar fields</mat-slide-toggle>
+      </div>
+    </mat-panel-title>
+  </mat-expansion-panel-header>
+  <app-panel-table
+    #commonPanel
+    [table]="common"
+    [tabIndex]="tabIndex"
+    [displayedColumns]="[
+      'column_indicator',
+      'column_name',
+      'column_type',
+      'comments'
+    ]"
+    (openTransform)="onOpenTransfromDialog($event)"
+  >
+  </app-panel-table>
+</mat-expansion-panel>
+
+<mat-expansion-panel (opened)="onOpen()" (closed)="onClose()" [expanded]="individual.expanded">
+  <mat-expansion-panel-header
+    #exppanelheader
+    class="mat-expansion-panel-header panel-header-{{ table.name }}"
+  >
+    <mat-panel-title>
+      <mat-icon
+        (click)="openSampleDataDialog($event)"
+        matTooltip="Open sample data"
+      >
+        list
+      </mat-icon>
+      <span>Individual</span>
+      <div class="linkSimilarItems">
+        <mat-slide-toggle
+          [disabled]="!individual.rows.length"
+          (change)="onCheckboxChange($event)"
+          (click)="$event.stopPropagation();"
+        >Link similar fields</mat-slide-toggle>
+      </div>
+    </mat-panel-title>
+  </mat-expansion-panel-header>
+  <app-panel-table
+    #individualPanel
+    [table]="individual"
+    [tabIndex]="tabIndex"
+    [displayedColumns]="[
```

Comment:
Let's move displayedColumns to readonly property with value ['column_indicator',  'column_name', 'column_type', 'comments']

mat-expansion-panel repeats for three times. The only difference is common, concept or individual keywords.
I suggest to make ngFor of ["common", "concept", "individual"]

Reply:
done